### PR TITLE
Add Area 51 to the list

### PR DESCRIPTION
**Pull request by Codeusa**

### CODE REVIEW

File: games.yaml [hunk 1]
```diff
@@ -174,6 +174,16 @@
       media:
         - image: http://xarchon.seul.org/img/snap-1.gif
         - image: http://xarchon.seul.org/img/snap-2.gif
+        
+- name: "Area 51 (2005, Midway)"
```
Comment (piranha) [inline games.yaml]: You have to write a name here which resolves to a page on Wikipedia. In your case I think it should be `[Area 51, Area 51 (2005 video game)]`.

Comment (Codeusa) [inline games.yaml]: @piranha my mistake, fixed.

**Comment by piranha** [comment]
Just a heads up: I went to the repository link you've provided, and didn't see any source there.

Admittedly, I should've done that before merging your pull request, but better later than never, right? Anyway, I've removed this entry for the time being. 

I'm open for discussion, though. :)

**Comment by Codeusa** [comment]
https://github.com/ProjectDreamland/launcher

https://github.com/ProjectDreamland/research/tree/master/game%20source

https://github.com/ProjectDreamland/xbmp-converter


The game is modified to work with an emulated gamespy network, I'm not seeing the issue. Source for everything in the project is there.

**Comment by piranha** [comment]
Excuse me, 24 Mb .c file? That looks like a result of decompilation, which is not an open source clone in any way.

**Comment by Codeusa** [comment]
Again, the game is modified to work with an emulated gamespy network, the file formats for unpacking and packing the game are present there as well. So are texture conversion tools for converting the custom texture formats.

Does it enable you to play an abandoned game online again? Yes. Does it allow you to modify the game? Yes. Can someone host their own? Yep. Does it fix a billion issues from the original, including security fixes? You bet it does.

I'm not really play semantics here, seems pointless and like a waste of time.

**Comment by piranha** [comment]
You seem to misunderstand purpose of this site. It's not about advertising games you can still play, it's about advertising *open source* work of people, so that other people can appreciate it and contribute to it.

While your work is admirable, is not an open source in any way. For example, I also do not accept Xash3d in the listing, which was done in similar fashion.

As for the waste of time - it is *you* wasting *my* time, since you submitted something which explicitly is not for this list, and now I'm spending time describing what you should be understanding from the start.

**Pull request merged**